src/components/common/TmBalance.vue

```diff
@@ -142,6 +202,45 @@ export default {
         return data.network.stakingDenom
       }
     },
+    totalFiatValue: {
+      query: gql`
+        query totalFiatValue(
```

Comment:
this produces a roundtrip. why not include this somehow in the overview. it will always be served together right?

Reply:
Didn't you say in the other PR you wanted to slice overview in separate queries?

Reply:
true, but the idea was to serve fiat values directly with any amount